mod errors;

use errors::*;

fn main() -> Result<()> {
    Ok(())
}
